### DISCUSSION TITLE
Is the automatic spins blueprint now safe to use?

### DISCUSSION THREAD

**Discussion by MaximumFish** [BottlecapDave/HomeAssistant-OctopusEnergy#920] [General]
In the release notes for 11.1.0 it says "Fixed spin wheel of fortune service" and I'm very much hoping it means that we can now safely use the auto-spins blueprint so that I don't have to sit through their little party popper animation for 1p of credit ever again!

Please tell me it's true. 😂

**Comment by BottlecapDave** [comment]
I'm afraid it was fixing it so the service could be executed. The service still doesn't seem to issue octoplus credit upon using and there doesn't seem to be any other service to achieve this via their standard APIs

**Reply by MaximumFish** [reply]
Dang it. Ah well, thanks for getting back to me.

**Comment by gcoan** [comment]
> I'm afraid it was fixing it so the service could be executed. The service still doesn't seem to issue octoplus credit upon using and there doesn't seem to be any other service to achieve this via their standard APIs

that's interesting to know.  I had setup the wheel of fortune automation but found I couldn't see any octopoints awarded so stopped using it and went back to using the wheel in the app

@BottlecapDave 
might it be worth highlighting this limitation in the automation example, or TBH, if it doesn't actually perform any customer benefit, discontinuing the wheel of fortune service?

**Reply by BottlecapDave** [reply]
It's already highlighted the docs as a warning with both the service and blueprint. The service gives you potential cash value if you're not on octoplus, so still has some value. I know of some users who use the integration who are not part of octoplus.
